Cloud Application Manager/Managed Services/getting-started-with-cam-create-managed-application.md

```diff
@@ -165,4 +165,4 @@ Learn to publish your new Managed Application script box to the public catalog.
 
 This step can be done in parallel to the previous step of creating the monitoring policy.  There is no need to wait for a response from MSA before publishing your script box.
 
-Publishing your script box is explained well at the following location: https://www.ctl.io/knowledge-base/cloud-application-manager/tutorials/publish-script-box/
+Publishing your script box is explained well [here](https://www.ctl.io/knowledge-base/cloud-application-manager/tutorials/publish-script-box/).
```

Comment:
Change to `(../Tutorials/publish-script-box.md)`
We run a process that verifies that the referenced KB article exists before we merge the PR. Then when building the site, these file paths are converted to links.